src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  overflow-x: scroll;
+}
+
+body {
+  position: relative;
+  height: 100%;
+  width: 100%;
+  max-width: 900px;
+  margin: auto;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.todolist {
+  margin: 20px 40px;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: -1px -1px 15px 2px rgba(192, 181, 181, 1);
+  border: 1px solid black;
+}
```

Comment:
[OPTIONAL] I think your design could be nicer if you tried to make it match the design of the to-do list you're cloning.
![image](https://user-images.githubusercontent.com/38088818/166098449-52d7ca91-45f0-4052-87a7-299f91a22a8e.png)

- You could use the grey color `#b2b2b2`  for your borders instead of `black` and you can try to center the todo to be in the middle of the page. As well as use the `bin` icon instead of `times`.
![image](https://user-images.githubusercontent.com/38088818/166098479-e2b9d699-a9dc-4e0e-a085-2de90985d9af.png)